lib/ansible/modules/cloud/amazon/elasticache.py

```diff
@@ -220,19 +222,19 @@ def create(self):
                 self.module.fail_json(msg=msg % self.name)
 
         try:
-            response = self.conn.create_cache_cluster(cache_cluster_id=self.name,
-                                                      num_cache_nodes=self.num_nodes,
-                                                      cache_node_type=self.node_type,
-                                                      engine=self.engine,
-                                                      engine_version=self.cache_engine_version,
-                                                      cache_security_group_names=self.cache_security_groups,
-                                                      security_group_ids=self.security_group_ids,
-                                                      cache_parameter_group_name=self.cache_parameter_group,
-                                                      cache_subnet_group_name=self.cache_subnet_group,
-                                                      preferred_availability_zone=self.zone,
-                                                      port=self.cache_port)
-        except boto.exception.BotoServerError as e:
-            self.module.fail_json(msg=e.message)
+            self.conn.create_cache_cluster(CacheClusterId=self.name,
+                                           NumCacheNodes=self.num_nodes,
+                                           CacheNodeType=self.node_type,
+                                           Engine=self.engine,
+                                           EngineVersion=self.cache_engine_version,
+                                           CacheSecurityGroupNames=self.cache_security_groups,
+                                           SecurityGroupIds=self.security_group_ids,
+                                           CacheParameterGroupName=self.cache_parameter_group,
+                                           CacheSubnetGroupName=self.cache_subnet_group,
+                                           PreferredAvailabilityZone=self.zone,
+                                           Port=self.cache_port)
+        except botocore.exceptions.ClientError as exception:
+            self.module.fail_json(msg=exception.response)
```

Comment:
Can you update this the exception handling in the guidelines, please?
 https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/cloud/amazon/GUIDELINES.md#boto3-2